arrows/serialize/protobuf/convert_protobuf.cxx

```diff
@@ -304,7 +304,7 @@ void convert_protobuf( const kwiver::vital::image_container_sptr img,
   const uLongf size = compressBound( local_image.size() );
   uLongf out_size(size);
   std::vector<uint8_t> image_data( size );
-  Bytef out_buf[size];
+  Bytef *out_buf = new Bytef[size];
```

Comment:
An alternate implementation would be to use a `std::unique<Bytef>` to manage the storage.